AppBidding-MoPub/AppBidding-MoPub/CriteoSample.h

```diff
@@ -0,0 +1,23 @@
+//
+//  CriteoSample.h
+//  AppBidding-MoPub
+//
+//  Copyright © 2020 Criteo. All rights reserved.
+//
+
+
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface CriteoSample : NSObject
+
+@property (assign) NSString *label;
+@property (assign) NSString *segueId;
+
+- (id) initWithLabel:(NSString *)label AndSegueId:(NSString *)segueId;
```

Comment:
Nit: AndSegueId should be renamed to andSegueId to match the naming conventions: https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/Conventions/Conventions.html